easybuild/easyconfigs/b/beagle-lib/beagle-lib-3.1.2-GCC-10.3.0-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '3.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/beagle-dev/beagle-lib'
+description = """beagle-lib is a high-performance library that can perform the core calculations at the heart of most
+ Bayesian and Maximum Likelihood phylogenetics packages."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/beagle-dev/beagle-lib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd872b484a3a9f0bce369465e60ccf4e4c0cd7bd5ce41499415366019f236275']
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Java', '17', '', True),
+    ('pkg-config', '0.29.2'),
+]
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('libtool', '2.4.6'),
+]
+
+preconfigopts = "sed -i 's/compute_30/compute_60/g' configure.ac && "
```

Comment:
rather than hardcoding `compute_60`, we should really be respecting the `cuda_compute_capabilities" specified

Reply:
Is it possible to do that in the EasyConfig or would I have to create an EasyBlock?